doc/bibliografia.bib

```diff
@@ -237,22 +237,22 @@ @misc{image-ansible
 	howpublished = {\url{http://blog.royalcyber.com/devops/new-level-of-automation-with-ansible/}}
 }
 
-@misc{,
-	author       = {}, 
-	title        = {},
-	howpublished = {\url{}}
+@misc{Netflix,
```

Comment:
No pongáis páginas web en la bibliografía, simplemente enlazadlas.

Reply:
done